cmd/atlas/atlas.go

```diff
@@ -108,15 +109,15 @@ func createConfigFromMongoCLIConfig() {
 
 func mongoCLIConfigFilePath() (configPath string, err error) {
 	if configDir, err := config.MongoCLIConfigHome(); err == nil {
-		configPath = fmt.Sprintf("%s/config.toml", configDir)
+		configPath = path.Join(configDir, "config.toml")
 	}
 
 	// Check if file exists, if any error is detected try to get older file
 	if _, err := os.Stat(configPath); err == nil {
 		return configPath, nil
 	}
 
-	if configDir, err := config.OldMongoCLIConfigHome(); err == nil {
+	if configDir, err := config.OldMongoCLIConfigHome(); err == nil { //nolint:staticcheck // check if old config exists
```

Comment:
had to add this when I enabled global linting again, the flagging the method as deprecated means you should not use it